README.md

```diff
@@ -43,6 +44,7 @@ Link for
 |message_title|title you want to add to the message|string|fluent_title_default
 |message|The message you want to send, can be a static message, which you add at this point, or you can receive the fluent infos with the %s|string|nil
 |enable_tls|you can set the communication channel if it uses tls|bool|true|
+|ca_path|you can set the path of the certificates|string|nil|
```

Comment:
weil hier steht "you can"
zudem hab ich keine Lust das anzugeben wenn ich self signed brauch

Reply:
Wenn du keinen Pfad gibst an, wird der Standardpfad verwendet. Wenn du bestimmte Zertifikate für MM verwenden willst, die sich an einem anderen Ort befinden, muss du den Pfad angeben.